python/cudf/cudf/_lib/includes/parquet.pxd

```diff
@@ -47,3 +47,29 @@ cdef extern from "cudf/cudf.h" namespace "cudf::io::parquet" nogil:
         cudf_table read_rows(size_t skip_rows, size_t num_rows) except +
 
         cudf_table read_row_group(size_t row_group) except +
+
+    cdef enum compression_type:
```

Comment:
This will make it an enum instead of an enum class under the hood. See this gist for how to get Cython to generate the enum class instead: https://gist.github.com/kkraus14/1ad8c5decabfd994633ef7f71dad3bc8